File: prak_lat1 python/Pc.py
from Processor import *
from Ram import *
from Disk import *

class Pc:
	# Constructor
	def __init__(self, processor, ram, disk):
		self.processor = processor
		self.ram = ram
		self.disk = disk
		self.totalPrice = processor.get_price() + ram.get_price() + disk.get_price()

	# Set & Get methods
	def set_processor(self, processor):
		self.processor = processor

	def set_ram(self, ram):
		self.ram = ram

	def set_disk(self, disk):
		self.disk = disk

	def set_total_price(self):
		self.totalPrice = processor.get_price() + ram.get_price() + disk.get_price()

	def get_processor(self):
		return self.processor
	
	def get_ram(self):
		return self.ram

	def get_disk(self):
		return self.disk

	def get_total_price(self):
		return self.totalPrice
		

File: prak_lat1 php/Pc.php
<?php

include "Processor.php";
include "Disk.php";
include "Ram.php";

class Pc{
	// Attributes
	private $totalPrice;
	private $pObj;
	private $dObj;
	private $rObj;

	// Constructor
	public function __construct($pObj, $dObj, $rObj){
		$this->pObj = $pObj;
		$this->dObj = $dObj;
		$this->rObj = $rObj;
		$this->totalPrice = $pObj->getPrice() + $dObj->getPrice() + $rObj->getPrice();
	}

	// Set & Get Methods
	public function setTotalPrice($pObj, $dObj, $rObj){
		$this->totalPrice = $pObj->getPrice() + $dObj->getPrice() + $rObj->getPrice();
	}

	public function setProcessor($pObj){
		$this->pObj = $pObj;
	}

	public function setDisk($dObj){
		$this->dObj = $dObj;
	}

	public function setRam($rObj){
		$this->rObj = $rObj;
	}

	public function getTotalPrice(){
		return $this->totalPrice;
	}

	public function getProcessor(){
		return $this->pObj;
	}

	public function getDisk(){
		return $this->dObj;
	}

	public function getRam(){
		return $this->rObj;
	}
	
}
?>